test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,11 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
-});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(24 , 20)).toEqual('I will be there in 1 hour');
```

Comment:
There is an extra space after 24 should be: reachDestination(24, 20)

And for better consistency I would use the "" double quotation marks, for "I will be there in 1 hour", and "I will be there in 2 hours" sentences